### ISSUE TITLE
BTT SKR Mini E3 1.2 with BTT Smart Filament Sensor an BLTouch ABL and TFT35 E3 3.0Possible?

### ISSUE THREAD

**Issue by Sn4k3E4t3r** [bigtreetech/BIGTREETECH-SKR-mini-E3#169]
Hi,

at first sorry for my bad english :) I am now the proud owner of the following products. I would like to connect the BTT Smart Filament Sensor and a BLTouch and a TFT35 E3 V3.0 to the SKR mini E3 V1.2 is that possible? I have an ender 3

**Comment by dziabi3** [comment]
Yes, you can connect, I have the same hardware configuration but I temporarily disconnected TFT 35 due to strange behavior.
It is possible that it was due to my ignorance. You will use this display again as soon as I get more experience from Marlin. I am currently devoting time to octoprint, which compensates me for the lack of TFT 35.

**Comment by demonkazuya** [comment]
> Yes, you can connect, I have the same hardware configuration but I temporarily disconnected TFT 35 due to strange behavior.
> It is possible that it was due to my ignorance. You will use this display again as soon as I get more experience from Marlin. I am currently devoting time to octoprint, which compensates me for the lack of TFT 35.

Could you please confirm what strange behavior you are having? I have the same setup but i'm unable to have the Filament sensor work. i have attached the sensor to PT-DET but every time I remove the filament the printer shuts down..
